src/modules/cmdpal/Microsoft.CmdPal.UI/Controls/SearchBar.xaml.cs

```diff
@@ -112,6 +123,23 @@ private void FilterBox_KeyDown(object sender, KeyRoutedEventArgs e)
 
     private void FilterBox_TextChanged(object sender, TextChangedEventArgs e)
     {
+        Debug.WriteLine($"FilterBox_TextChanged: {FilterBox.Text}");
+
+        // TERRIBLE HACK
+        // There's weird wacky bugs with the debouncer currently. We're trying
+        // to get them ingested, but while we wait for the toolkit feeds to
+        // bubble, just manually send the first character, always
+        // (otherwise aliases just stop working)
+        if (FilterBox.Text.Length == 1)
+        {
+            if (CurrentPageViewModel != null)
+            {
+                CurrentPageViewModel.Filter = FilterBox.Text;
+            }
+
+            return;
+        }
```

Comment:
Would you mind at least trying to pull from Main Latest locally? Just to at least confirm that the behavior is fixed?

```
https://pkgs.dev.azure.com/dotnet/CommunityToolkit/_packaging/CommunityToolkit-MainLatest/nuget/v3/index.json
```

If not, then I could at lest ensure we fix it in the Toolkit again if there is still an issue before we ship the next update to NuGet.

Reply:
@ me go get rid of the other HACK from #229 too

Reply:
It looks like with `CommunityToolkit.WinUI.Extensions" Version="8.2.241204-build.1297`, I could delete both these HACK sections out of this file. So that's good! I'll file a TODO to update to the latest toolkit version when that's available and get rid of these